test/extended/networking/services.go

```diff
@@ -0,0 +1,67 @@
+package networking
+
+import (
+	"k8s.io/kubernetes/test/e2e"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("[networking] services", func() {
+	f1 := e2e.NewFramework("net-isolation1")
+	f2 := e2e.NewFramework("net-isolation2")
+
+	It("should allow connections to other pods via service IPs", func() {
```

Comment:
Consider skipping this test instead of failing (via getMultipleNodes) if only one node is found to be present.  The kube e2e service conformance check will effectively validate single node service connectivity in that case:

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/service.go#L82